pkg/storegateway/bucket_store_metrics.go

```diff
@@ -96,6 +97,10 @@ func NewBucketStoreMetrics(reg prometheus.Registerer) *BucketStoreMetrics {
 		Name: "cortex_bucket_store_series_blocks_queried",
 		Help: "Number of blocks in a bucket store that were touched to satisfy a query.",
 	})
+	m.seriesBlockSourceAndLevelQueried = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+		Name: "cortex_bucket_store_series_blocks_queried_by_source_and_level",
```

Comment:
my 2c on the metric name is that having `by_source_and_level` kind of goes against the prometheus conventions. I know we already have one metric ending in `by_duration`, but this naming prevents from adding new divisions later on and repeats what the labels on the metric already say. 


So my choice would have been to add the two labels on the existing `cortex_bucket_store_series_blocks_queried`.

Reply:
Good suggestion. Done.